File: go/exercise07/area_of_rectanglar_room.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"strconv"
)

func main() {
	scanner := bufio.NewScanner(os.Stdin)

	fmt.Print("What is the length of room in feet? ")
	scanner.Scan()
	length, _ := strconv.Atoi(scanner.Text())

	fmt.Print("What is the width of room in feet? ")
	scanner.Scan()
	width, _ := strconv.Atoi(scanner.Text())

	squareMeters := (float64(length) * float64(width) * 0.09290304)
	fmt.Printf("You entered dimensions of %v feet by %v feet.\n", length, width)
	fmt.Printf("The area is\n%v square feet\n%v square meters\n", length*width, squareMeters)
}

File: c/area_of_rectanglar_room.c
#include <stdio.h>
#include <stdlib.h>

int main(void) {
  float length = 0;
  float width = 0;

  printf("What is the length of room in feet? ");
  scanf("%f", &length);

  printf("What is the width of room in feet? ");
  scanf("%f", &width);

  float square_meters = (length * width) * 0.09290304;
  printf("You entered dimensions of %.2f feet by %.2f feet.\n", length, width);
  printf("The area is\n%.2f square feet\n%.3f square meters\n", length * width,
         square_meters);

  return EXIT_SUCCESS;
}
